processor/ic/src/sources/mod.rs

```diff
@@ -0,0 +1,68 @@
+use pyth::Pyth;
+use redstone::Redstone;
+use verity_dp_ic::verify::types::{ProofResponse, ProofTypes, VerificationCanisterResponse};
```

Comment:
**A note:** the `ProofResponse` is indeed correct to be in Verity DP, as it's a re-purposeable primitive for parsing TLS proofs between projects that implement the Verity DP.